### PR TITLE
feature: Add artists filter to gene page

### PR DESCRIPTION
**Pull request by olerichter00**
Resolves [FX-2859]

[FX-2859]: https://artsyproduct.atlassian.net/browse/FX-2859

## Screenshots

image
image

image

### CODE REVIEW

File: src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx [hunk 1]
```diff
@@ -109,6 +120,7 @@ export const ArtistsFilter: FC<ArtistsFilterProps> = ({
           onSelect={value =>
             filterContext.setFilter("includeArtworksByFollowedArtists", value)
           }
+          my={tokens.my}
```
Comment (olerichter00) [inline src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx:123]: Note: we need to add the padding here (e.g. like here https://github.com/artsy/force/blob/e9842c2ab8b39c32812ee3f1b344553cae63a72d/src/v2/Components/v2/ArtworkFilter/ArtworkFilters/MediumFilter.tsx#L66)

File: src/v2/Apps/Gene/Components/GeneArtworkFilter.tsx [hunk 2]
```diff
@@ -16,11 +36,44 @@ const GeneArtworkFilter: React.FC<GeneArtworkFilterProps> = ({
...
 }) => {
   const { match } = useRouter()
+  const {
+    filtered_artworks: { counts },
+    sidebarAggregations,
+  } = gene
+
+  const { relayEnvironment, user } = useSystemContext()
+
+  const showNewFilters = getENV("ENABLE_NEW_ARTWORK_FILTERS")
```
Comment (dzucconi) [inline src/v2/Apps/Gene/Components/GeneArtworkFilter.tsx:46]: You can safely ignore this flag now. There's actually a ticket to get rid of it completely in the other components.

Comment (olerichter00) [reply src/v2/Apps/Gene/Components/GeneArtworkFilter.tsx:46]: 👍 I removed the flag - now showing the new filter.

File: src/v2/Apps/Gene/Components/GeneArtworkFilter.tsx [hunk 3]
```diff
@@ -16,11 +36,44 @@ const GeneArtworkFilter: React.FC<GeneArtworkFilterProps> = ({
...
+
+  // TODO: fix me - the ArtistsFilter will fetch followed artists only for fairs
```
[the remark this answers is not in the document]
Comment (olerichter00) [reply src/v2/Apps/Gene/Components/GeneArtworkFilter.tsx:48]: In `ArtistsFilter` we use a method to fetch the user's followed artists which requires a fair-id to work (the component was used for the fare artwork page only). 

https://github.com/artsy/force/blob/master/src/v2/Components/v2/ArtworkFilter/Utils/fetchFollowedArtists.ts

**Comment by damassi** [comment]
@olerichter00 - can this PR be closed now that the work has been completed elsewhere?

**Comment by dzucconi** [comment]
@damassi this facet hasn't been added yet (AFAIK?) — I'm a little unclear on what this needs to get it over the line; but it remains a ticket on FX's board.

**Comment by damassi** [comment]
Oh, forgive me; I read "artists fillter" as "artworks filter" -- thinking that this page was work to migrate /gene/id to the new theme. I see now that this is a new singular filter 👍

**Review by dzucconi — APPROVED** [review]
🙏

**Comment by olerichter00** [comment]
🙏

**Pull request merged**